nextcord/polls.py

```diff
@@ -0,0 +1,437 @@
+class PollAnswer:
+    """
+    A choice to answer in a poll.
+
+    .. versionadded:: 3.0
+
+    Attributes
+    ----------
+    answer_id: Optional[:class:`int`]
+        The answer ID of the answer.
+    poll_media: Optional[:class:`PollMedia`]
+        The poll media of the answer.
+    """
+
+    __slots__ = (
+        "answer_id",
+        "poll_media",
+    )
+
+    def __repr__(self) -> str:
+        return f"<PollAnswer answer_id={self.answer_id!r} poll_media={self.poll_media!r}>"
+
+    def __init__(self, *, answer_id: Optional[int] = None, poll_media: PollMedia) -> None:
+        self.answer_id: Optional[int] = answer_id
+        self.poll_media: PollMedia = poll_media
+
+    def to_dict(self) -> PollAnswerPayload:
+        payload: PollAnswerPayload = {"poll_media": self.poll_media.to_dict()}
+        if self.answer_id:
+            payload["answer_id"] = self.answer_id
+
+        return payload
+
+    @classmethod
+    def from_dict(cls, data: PollAnswerPayload) -> Self:
+        # data['answer_id'] is always sent as part of Discord's HTTP/Gateway
+        return cls(answer_id=data["answer_id"], poll_media=PollMedia.from_dict(data["poll_media"]))  # type: ignore[reportTypedDictNotRequiredAccess]
+
+
+class PollAnswerCount:
+    """The answer count for an answer in a poll.
+
+    .. versionadded:: 3.0
+
+    Attributes
+    ----------
+    id: :class:`int`
+        The ID of the answer in question.
+    me_voted: :class:`bool`
+        Whether the bot has voted for this option.
+    count: :class:`int`
+        The number of users who have voted for this option.
+    """
+
+    __slots__ = ("id", "me_voted", "count", "poll")
+
+    def __repr__(self) -> str:
+        return (
+            "<PollAnswerCount "
+            f"id={self.id!r} "
+            f"me_voted={self.me_voted!r} "
+            f"count={self.count!r} "
+            ">"
+        )
+
+    def __init__(self, data: PollAnswerCountPayload, poll: Poll) -> None:
+        self.poll: Poll = poll
+        self.id: int = data["id"]
+        self.me_voted: bool = data["me_voted"]
+        self.count: int = data["count"]
+
+    def voters(
+        self, *, limit: Optional[int] = None, after: Optional[Snowflake] = None
+    ) -> AnswerVotersIterator:
+        """Returns an :class:`AsyncIterator` representing the users who have voted for this option.
+
+        The ``after`` parameter must represent an user
+        and meet the :class:`abc.Snowflake` abc.
+
+        Examples
+        --------
+        Usage ::
+
+            # I do not actually recommend doing this.
+
+            async for voter in count.voters():
+                await channel.send(f"{voter} has voted with choice number {count.id}!")
+
+        Flattening into a list ::
+
+            voters = await count.voters().flatten()
+            # voters is now a list of User...
+            await channel.send("Those people have voted with choice number {count.id}: {', '.join(voters)}")
+
+        .. versionadded:: 3.0
```

Comment:
This `versionadded` is unnecessary since the whole object is new since version 3.0, and you've already specified it above.